src/AstroBase.jl

```diff
@@ -1,5 +1,125 @@
 module AstroBase
 
 # package code goes here
+export xy06, fal03, falp03, faf03, fad03, faom03, fame03, fave03, fae03, fama03, faju03, fasa03,
+       faur03, fane03, fapa03
 
+include("mfals.jl")
+
+const J2000 = 2451545.0
+const DAYS_PER_CENTURY = 36525.0
+const TURNAS = 1296000.0
+
+sec2rad(sec::Real) = deg2rad(sec/3600)
+
+fal03(t)  =  mod((@evalpoly t 485868.249036 1717915923.2178 31.8792 0.051635 -0.00024470) , TURNAS) * sec2rad(1)
+falp03(t) =  mod((@evalpoly t 1287104.793048 129596581.0481 -0.5532 0.000136 -0.00001149),  TURNAS) * sec2rad(1)
+faf03(t)  =  mod((@evalpoly t 335779.526232 1739527262.8478 -12.7512 -0.001037 0.00000417), TURNAS) * sec2rad(1)
+fad03(t)  =  mod((@evalpoly t 1072260.703692 1602961601.2090 -6.3706 0.006593 -0.00003169), TURNAS) * sec2rad(1)
+faom03(t) =  mod((@evalpoly t 450160.398036 -6962890.5431 7.4722 0.007702 -0.00005939), TURNAS ) * sec2rad(1)
+fame03(t) =  mod2pi(4.402608842 + 2608.7903141574t)
+fave03(t) =  mod2pi(3.176146697 + 1021.3285546211t)
+fae03(t)  =  mod2pi(1.753470314 + 628.3075849991t)
+fama03(t) =  mod2pi(6.203480913 + 334.0612426700t)
+faju03(t) =  mod2pi(0.599546497 + 52.9690962641t)
+fasa03(t) =  mod2pi(0.874016757 + 21.3299104960t)
+faur03(t) =  mod2pi(5.481293872 + 7.4781598567t)
+fane03(t) =  mod2pi(5.311886287 + 3.8133035638t)
+fapa03(t) =  (0.024381750 + 0.00000538691t) * t
+
+function xy06(jd1, jd2)
+    NFLS, NFPL, NA, MAXPT= length(mfals), length(mfapl), length(amp), 5
```

Comment:
I find this very hard to read and would suggest to give every assignment its own line.

Reply:
sure!